src/agent/onefuzz-agent/Cargo.toml

```diff
@@ -16,7 +16,7 @@ async-trait = "0.1"
 clap = "2.33"
 tempfile = "3.1"
 env_logger = "0.7"
-futures = "0.3"
+futures = "0.3.7"
```

Comment:
updated because of an issue reported by cargo audit:
```
Crate:         futures-util
Version:       0.3.6
Title:         MutexGuard::map can cause a data race in safe code
Date:          2020-10-22
ID:            RUSTSEC-2020-0059
URL:           https://rustsec.org/advisories/RUSTSEC-2020-0059
Solution:      Upgrade to >=0.3.7
```

Reply:
I had to merge this change as part of the cargo-audit fix #242.